FinalGPTProj/Client.py

```diff
@@ -0,0 +1,69 @@
+import time
+import requests
+
+class Status:
+    def __init__(self, status, filename, timestamp, explanation):
+        self.status = status
+        self.filename = filename
+        self.timestamp = timestamp
+        self.explanation = explanation
+
+    def is_done(self):
+        return self.status == "done"
+
+class GPTExplainerClient:
+    def __init__(self, base_url):
+        self.base_url = base_url
+
+    def upload(self, file_path):

```

Comment:
Write documentation any of the standart python documentation formats, here is an example of one -
```python
def count_hi_v4(word: str) -> int:
"""
The function counts how many times the string "hi" appears in a given string.
@param word: The given string.
@return: The amount of times the word "hi" appeared.
"""
return word.count("hi")
```